SYSTEM/Settimera.md

```diff
@@ -8,7 +8,7 @@ ns: SYSTEM
 void SETTIMERA(int value);
 ```
 
-
 ## Parameters
-* **value**: 
+* **Time**: In milliseconds.
```

Comment:
Hey, rename this to `value`, it needs to match the function argument declared on the function declaration.